### ISSUE TITLE
Allow specifing a JSON file path with options for `wp core install` command

### ISSUE THREAD

**Issue by vadimkantorov** [wp-cli/core-command#278]
Could be a convenience for specifying directly a JSON file with exported options to the `wp core install command`. Related to:
- https://github.com/wp-cli/entity-command/issues/512 - current situation is that it's impossible or very hard to solve all string escaping issues to batch-import all options from the JSON file

I suggest that this would initialize the `wp_options` table from the values in the JSON file. The JSON file format should be the same as used in `wp option list --format=json` command output. This would simplify usage of WordPress as a SSG engine (experimented with at https://github.com/vadimkantorov/wpssg)

**Comment by mrsdizzie** [comment]
If you want a way to copy options from one site to another now:
```
wp db export --tables=wp_options --no-create-info=true --replace=true options.sql

wp db import options.sql
```
Which will give you a sql dump file that you can import to another site to replace any existing options with the previous ones.

**Comment by vadimkantorov** [comment]
In my SSG hack I am trying to keep the WP content in common human-editable files (like markdown for posts and JSON for options. I also wonder if the block editor in wordpress comes with its own markup language spec?).

SQL format for options files also would work, but for more convenience JSON would be preferrable (as it's also the most basic format implemented in de/serialization in all programming languages and easy for working both manually and with any automation)

**Comment by mrsdizzie** [comment]
Thats going to be outside the scope of wp cli I think (just to set expectations and that nobody is waiting on this to be implemented).

The current JSON output from wp cli commands is there for easier parsing of output and isn't intended as a way to save data and pass it back and forth from WordPress installs since WordPress core has no support for that. SQL should be used for this purpose unless you want to write your own importer to handle JSON.

**Comment by vadimkantorov** [comment]
`wp option update` has some support for importing JSON values but I failed to surmount the string escaping problems in 
- https://github.com/wp-cli/entity-command/issues/512
when I tried to make it work...

**Comment by swissspidy** [comment]
Closing in favor of https://github.com/wp-cli/entity-command/issues/512 since import/export functionaltiy is out of scope for `wp core install`. That command is there solely for running the install process, nothing more.
